skglm/experimental/pdcd_ws.py

```diff
@@ -9,33 +9,42 @@
 
 
 class PDCD_WS:
-    """Primal-Dual Coordinate Descent solver with working sets.
+    r"""Primal-Dual Coordinate Descent solver with working sets.
 
-    Solver inspired by [1] that uses working sets [2].
+    It solves::
+
+        \min_w F(Xw) + G(w) \Leftrightarrow \min_w \max_z <Xw, z> + G(w) - F^*(z)
+
+    where :math:`F` is the datafit term (:math:`F^*` it's Fenchel conjugate)
+    and :math:`G` is the penalty term.
+
+    The datafit is required to be convex and proximable. Also, the penalty
+    is also required to be convex, separable, and proximable.
+
+    The solver is inspired by [1] and uses working sets [2].
```

Comment:
it's more than inspired by [1]: The solver is the primal dual coordinate descent solver of ;[1], combined with working sets [2]